SPECS/influxdb/influxdb.spec

```diff
@@ -17,11 +17,11 @@
 
 Summary:        Scalable datastore for metrics, events, and real-time analytics
 Name:           influxdb
-Version:        2.6.1
-Release:        12%{?dist}
+Version:        2.7.5
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Distribution:   Microsoft Azure Linux
```

Comment:
Same comment.  Ok to drop Microsoft here

Reply:
Fixed